### ISSUE TITLE
XSS Reorganization

### ISSUE THREAD

**Issue by ThunderSon** [OWASP/wstg#591]
**What's the current setup?**
Reviewing XSS overall in the document, it's split into 2 different chapters, 7 and 11.
At the end of the day, all XSS will target users on the client-side, even if it was stored.

**What would you like to happen?**
Move all XSS to chapter 11, client-side testing, and have one big section for XSS and then going into more tests for other types of testing.
What do you guys think?

**Comment by Magii204** [comment]
I would like to be  work on this.

**Comment by linzinha** [comment]
Hows this for order? Unless you just wanted 7 added to the end of chapter 11

> 01-Testing_for_Reflected_Cross_Site_Scripting.md
02-Testing_for_Stored_Cross_Site_Scripting.md
03-Testing_for_HTTP_Verb_Tampering.md
04-Testing_for_HTTP_Parameter_Pollution.md
15-Testing_for_HTTP_Splitting_Smuggling.md
16-Testing_for_HTTP_Incoming_Requests.md
08-Testing_for_SSI_Injection.md
12-Testing_for_Command_Injection.md
13-Testing_for_Buffer_Overflow.md
13-Testing_for_Format_String_Injection.md
14-Testing_for_Incubated_Vulnerability.md
18-Testing_for_Server-side_Template_Injection.md
19-Testing_for_Server-Side_Request_Forgery.md
10-Testing_for_IMAP_SMTP_Injection.md
07-Testing_for_XML_Injection.md
09-Testing_for_XPath_Injection.md
05-Testing_for_SQL_Injection.md
05.1-Testing_for_Oracle.md
05.2-Testing_for_MySQL.md
05.3-Testing_for_SQL_Server.md
05.4-Testing_PostgreSQL.md
05.5-Testing_for_MS_Access.md
05.6-Testing_for_NoSQL_Injection.md
05.7-Testing_for_ORM_Injection.md
05.8-Testing_for_Client-side.md
11-Testing_for_Code_Injection.md
11.1-Testing_for_File_Inclusion.md
02-Testing_for_JavaScript_Execution.md
03-Testing_for_HTML_Injection.md
05-Testing_for_CSS_Injection.md
08-Testing_for_Cross_Site_Flashing.md
09-Testing_for_Clickjacking.md
10-Testing_WebSockets.md
11-Testing_Web_Messaging.md
12-Testing_Browser_Storage.md
13-Testing_for_Cross_Site_Script_Inclusion.md
04-Testing_for_Client-side_URL_Redirect.md
06-Testing_for_Client-side_Resource_Manipulation.md
01-Testing_for_DOM-based_Cross_Site_Scripting.md
01.1-Testing_for_Self_DOM_Based_Cross_Site_Scripting.md
07-Testing_Cross_Origin_Resource_Sharing.md
17-Testing_for_Host_Header_Injection.md
06-Testing_for_LDAP_Injection.md

**Comment by linzinha** [comment]
So, and also just to be clear then, would it just be the DOM section?

These seem to fit pretty well
4.11.2 Testing for JavaScript Execution
4.11.13 Testing for Cross Site Script Inclusion

These as well, though with less confidence:
4.11.7 Testing Cross Origin Resource Sharing
4.11.8 Testing for Cross Site Flashing
4.11.9 Testing for Clickjacking

**Comment by linzinha** [comment]
I deleted my last comment, was looking at the wrong tab!

**Comment by kingthorin** [comment]
Those are already in 11 👍

**Comment by linzinha** [comment]
Yeah, there was a sudden bit of chaos in the house and I realized with the distraction I'd been looking at 11 not 7

**Comment by kingthorin** [comment]
All good, been there.

**Comment by kingthorin** [comment]
Ya that can stay as-is.

**Comment by manindar-mohan** [comment]
I will work on this
